package.json

```diff
@@ -38,7 +38,7 @@
     "jasmine": "2.3.1",
     "jsdom": "5.6.1",
     "jsx-loader": "0.12.2",
-    "lerna": "2.0.0-beta.23",
+    "lerna": "git://github.com/lerna/lerna.git#master",
```

Comment:
build is passing locally with my lerna built off master (which runs `prepublish` as part of bootstrap), but not on ci so I tried pulling from master.  However, lerna's master branch isn't a functioning node module, so this isn't working either.  Next, I'll try a `lerna run prepublish` on ci, but if that doesn't work, we may just need to wait for lerna-2.0.0-beta.24